src/toast/ToastContainer.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as ReactDOM from "react-dom";
+import * as React from "react";
+import Toast from "./Toast";
+
+const ID = "toast-container";
+
+class ToastContainer {
```

Comment:
I think you can club this ToastContainer class code in 'Toast' class itself. Please refer code for 'Modal' component.

Reply:
I don't think this class is needed at all and we need to follow standard React programming style.  Any class we use should implement React component.  We also should not be accessing the DOM directly and instead utilize React methods.

Reply:
Updated code.